Controller/Adminhtml/Account/Proxy.php

```diff
@@ -56,6 +59,7 @@ public function __construct(
     ) {
         parent::__construct($context);
 
+        $this->_publicActions = ['proxy'];
```

Comment:
Would it make sense to move into a constant?

Reply:
Comes from a Magento base class, better not to touch it :)